src/FirebaseObject.h

```diff
@@ -21,14 +21,49 @@
 
```

Comment:
Since this is a little messy and really just to test our ReadTheDocs I would rather just keep it separate.